docs/topics/native/native-memory-manager.md

```diff
@@ -141,7 +162,7 @@ update might improve memory consumption.
 
 If you continue to experience high memory consumption after updating, several options are available:
 
-* Switch to a different memory allocator by using one of the following compilation options in your Gradle build script:
+* Switch to a different memory allocator by using one of the following compiler option in your Gradle build script:
 
   * `-Xallocator=mimalloc` for the [mimalloc](https://github.com/microsoft/mimalloc) allocator.
```

Comment:
Not quite sure how it should be done properly, but please note that we plan to deprecate the mimalloc allocator in 2.1.0. Perhaps, we could stop suggesting as the first workaround option it already now?

Reply:
thank you! I put the standard allocator option on the top. Please check if there's a "kotlin to be documented" tag on the mimalloc deprecation task in YT, so that we don't forget it in 2.1